pasta_eln/serverActions.py

```diff
@@ -486,7 +486,7 @@ def restoreCouchDB(location:str='', userName:str='', password:str='', fileName:s
     #second run through: create attachments
     for fileI in files:
       fileParts = fileI.split('/')[1:]
-      if fileParts==['pastaELN.json']: #do not recreate file, it is only there for manual recovery
+      if fileParts==['pastaELN.json'] or fileParts[-1]=='' or fileParts[-2]=='_design': #do not recreate file, it is only there for manual recovery
```

Comment:
When is fileParts[-1]=='' happening? This means the last part is an empty string, what kind of file is it?

Reply:
Those are folders, as these end with a '/' and should be skipped

Reply:
Hanna saw, that these exist in her files and caused an warning, which should not be raised.

Reply:
@jmurugan-fzj :+1: I corrected the code accordingly and files only holds files.